ds-card-reader/Dockerfile

```
@@ -1,14 +1,15 @@
-# Copyright © 2022 Intel Corporation. All rights reserved.
+# Copyright © 2023 Intel Corporation. All rights reserved.
 # SPDX-License-Identifier: BSD-3-Clause
 
-FROM golang:1.18-alpine3.16 AS builder
+FROM golang:1.20-alpine3.17 AS builder
 
 LABEL license='SPDX-License-Identifier: BSD-3-Clause' \
-  copyright='Copyright (c) 2022: Intel'
+  copyright='Copyright (c) 2023: Intel'
 
 
 # add git for go modules
-RUN apk update && apk add --no-cache make git gcc libc-dev zeromq-dev linux-headers
+# hadolint ignore=DL3018
+RUN apk update && apk add --no-cache make git gcc libc-dev linux-headers
```

```suggestion
RUN apk update && apk add --no-cache make git 
```

do we still need those c related dependency libs?

---

I've tried to remove them, docker build gave me error, so I still need them

---

device needs gcc

---

Yep, i think this service uses a module that requires CGO and gcc